runtime/ms-rest-azure/package.json

```diff
@@ -29,7 +29,8 @@
     "uuid": "^3.2.1",
     "adal-node": "^0.1.27",
     "ms-rest": "^2.3.2",
-    "moment": "^2.20.1"
+    "moment": "^2.20.1",
+    "request": "^2.83.0"
```

Comment:
This should not be the case. What version of npm are you using? 
`ms-rest-azure -(depends-on)-> ms-rest -(depends-on)->request`. Newer versions of npm dedupe/flatten the dependency structure. Are you using request somewhere in your app? If so then it may have a different version which would be colliding with the version ms-rest depends on. As a result npm may be installing ms-rest inside ms-rest-azure, or request under ms-rest, rather than a top level folder inside node_modules.